### ISSUE TITLE
Scroll jumps after autoresize if parent has scroll

### ISSUE THREAD

**Issue by maksimr** [jackmoore/autosize#299]
UC:

* Create element which contains autoresized textarea
* Add scroll and some content after textarea
* Start typing in textarea

**ER**: scroll does not jump
**AR**: scroll is jump

**Example which reproduce the problem**: https://jsfiddle.net/oy3poxbr/3/

Similar problem: https://github.com/jackmoore/autosize/issues/286

Thanks

**Comment by dgrad** [comment]
Running into this problem too.

Discussions here might help:
https://github.com/angular/material/issues/3070#issuecomment-108098128
(same issue being seen on angular material)

**Comment by Boberro** [comment]
Their code seems to be similar to what's done in autoresize, but simple port doesn't work.

**Comment by jacksondc** [comment]
Looks like there's a pull request with a patch (#298), but it's not quite working yet.

I got a fix working using #292, which adds a hook before the resize triggers:

````js
var scrollHeight;

autosizedElement.addEventListener('autosize:resizing', function() {
    scrollHeight = scrollingParentElement.scrollTop;
});

autosizedElement.addEventListener('autosize:resized', function() {
    scrollingParentElement.scrollTop = scrollHeight;
}
````

Not ideal but it works well enough.

**Comment by jackmoore** [comment]
Fixed with v3.0.16.  Sorry for the late response everyone.
